components/scream/src/physics/spa/spa_functions_impl.hpp

```diff
@@ -599,7 +608,7 @@ ::update_spa_data_from_file(
   Kokkos::deep_copy(target_grid_loc_h, spa_horiz_interp.target_grid_loc);
   for (int idx=0;idx<spa_horiz_interp.length;idx++) {
```

Comment:
Unrelated, but why are we doing this big interp loop on host? This seems like a beefy loop, so might benefit from device acceleration. Although, being done only once per month (right?), we should prob check how much it costs first.